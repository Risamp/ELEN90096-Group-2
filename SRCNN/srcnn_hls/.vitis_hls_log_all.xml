<?xml version="1.0" encoding="UTF-8"?>
<vivadoHLSLog:LogRoot xmlns:vivadoHLSLog="www.xilinx.com/vivadoHLSLog">
  <errorLogs>
    <logs>
      <synLog/>
      <simLog/>
      <mgLog/>
      <packageLog/>
      <csimLog/>
    </logs>
  </errorLogs>
  <warningLogs>
    <logs>
      <synLog>
        <logs message="WARNING: [RTGEN 206-101] Design contains AXI ports. Reset is fixed to synchronous and active low." projectName="srcnn_hls" solutionName="solution1" date="2023-11-02T02:40:22.194+1100" type="Warning"/>
        <logs message="WARNING: [RTGEN 206-101] Setting dangling out port 'conv3/grp_fu_284_p_opcode' to 0." projectName="srcnn_hls" solutionName="solution1" date="2023-11-02T02:40:20.509+1100" type="Warning"/>
        <logs message="WARNING: [HLS 200-2049] Adding the rewind optimization to the pipelined process 'RELU' with inout ports inside a dataflow region can result in the creation of loop-carried dependencies and errors. Use caution when adding the rewind optimization to such loops." projectName="srcnn_hls" solutionName="solution1" date="2023-11-02T02:40:20.056+1100" type="Warning"/>
        <logs message="WARNING: [HLS 200-2049] Adding the rewind optimization to the pipelined process 'RELU' with inout ports inside a dataflow region can result in the creation of loop-carried dependencies and errors. Use caution when adding the rewind optimization to such loops." projectName="srcnn_hls" solutionName="solution1" date="2023-11-02T02:40:19.656+1100" type="Warning"/>
        <logs message="WARNING: [HLS 200-2049] Adding the rewind optimization to the pipelined process 'RELU' with inout ports inside a dataflow region can result in the creation of loop-carried dependencies and errors. Use caution when adding the rewind optimization to such loops." projectName="srcnn_hls" solutionName="solution1" date="2023-11-02T02:40:16.981+1100" type="Warning"/>
        <logs message="WARNING: [HLS 200-2049] Adding the rewind optimization to the pipelined process 'RELU' with inout ports inside a dataflow region can result in the creation of loop-carried dependencies and errors. Use caution when adding the rewind optimization to such loops." projectName="srcnn_hls" solutionName="solution1" date="2023-11-02T02:40:16.646+1100" type="Warning"/>
        <logs message="WARNING: [RTGEN 206-101] Setting dangling out port 'conv1/grp_fu_284_p_opcode' to 0." projectName="srcnn_hls" solutionName="solution1" date="2023-11-02T02:40:15.452+1100" type="Warning"/>
        <logs message="WARNING: [RTGEN 206-101] Setting dangling out port 'export_output_buffer_c1/grp_fu_1352_p_opcode' to 0." projectName="srcnn_hls" solutionName="solution1" date="2023-11-02T02:40:14.490+1100" type="Warning"/>
        <logs message="WARNING: [HLS 200-2049] Adding the rewind optimization to the pipelined process 'RELU' with inout ports inside a dataflow region can result in the creation of loop-carried dependencies and errors. Use caution when adding the rewind optimization to such loops." projectName="srcnn_hls" solutionName="solution1" date="2023-11-02T02:40:14.141+1100" type="Warning"/>
        <logs message="WARNING: [HLS 200-2049] Adding the rewind optimization to the pipelined process 'RELU' with inout ports inside a dataflow region can result in the creation of loop-carried dependencies and errors. Use caution when adding the rewind optimization to such loops." projectName="srcnn_hls" solutionName="solution1" date="2023-11-02T02:40:13.850+1100" type="Warning"/>
        <logs message="WARNING: [HLS 200-960] Cannot flatten loop 'TILE_ROW' (src/conv1.cpp:31:12) in function 'conv1' more than one sub loop.&#xD;&#xA;Resolution: For help on HLS 200-960 see www.xilinx.com/cgi-bin/docs/rdoc?v=2023.1;t=hls+guidance;d=200-960.html" projectName="srcnn_hls" solutionName="solution1" date="2023-11-02T02:40:00.682+1100" type="Warning"/>
        <logs message="WARNING: [HLS 200-960] Cannot flatten loop 'TILE_OUT' (src/conv1.cpp:35:13) in function 'conv1' more than one sub loop.&#xD;&#xA;Resolution: For help on HLS 200-960 see www.xilinx.com/cgi-bin/docs/rdoc?v=2023.1;t=hls+guidance;d=200-960.html" projectName="srcnn_hls" solutionName="solution1" date="2023-11-02T02:40:00.212+1100" type="Warning"/>
        <logs message="WARNING: [HLS 200-960] Cannot flatten loop 'COL' (src/conv1.cpp:43:10) in function 'conv1' the outer loop is not a perfect loop because there is nontrivial logic in the loop latch.&#xD;&#xA;Resolution: For help on HLS 200-960 see www.xilinx.com/cgi-bin/docs/rdoc?v=2023.1;t=hls+guidance;d=200-960.html" projectName="srcnn_hls" solutionName="solution1" date="2023-11-02T02:40:00.194+1100" type="Warning"/>
        <logs message="WARNING: [HLS 200-960] Cannot flatten loop 'TILE_ROW' (src/conv2.cpp:31:12) in function 'conv2' more than one sub loop.&#xD;&#xA;Resolution: For help on HLS 200-960 see www.xilinx.com/cgi-bin/docs/rdoc?v=2023.1;t=hls+guidance;d=200-960.html" projectName="srcnn_hls" solutionName="solution1" date="2023-11-02T02:40:00.179+1100" type="Warning"/>
        <logs message="WARNING: [HLS 200-960] Cannot flatten loop 'TILE_OUT' (src/conv2.cpp:35:13) in function 'conv2' more than one sub loop.&#xD;&#xA;Resolution: For help on HLS 200-960 see www.xilinx.com/cgi-bin/docs/rdoc?v=2023.1;t=hls+guidance;d=200-960.html" projectName="srcnn_hls" solutionName="solution1" date="2023-11-02T02:40:00.171+1100" type="Warning"/>
        <logs message="WARNING: [HLS 200-960] Cannot flatten loop 'EXPORT' (src/conv2.cpp:104:10) in function 'conv2' the outer loop is not a perfect loop because there is nontrivial logic before entering the inner loop.&#xD;&#xA;Resolution: For help on HLS 200-960 see www.xilinx.com/cgi-bin/docs/rdoc?v=2023.1;t=hls+guidance;d=200-960.html" projectName="srcnn_hls" solutionName="solution1" date="2023-11-02T02:40:00.161+1100" type="Warning"/>
        <logs message="WARNING: [HLS 200-960] Cannot flatten loop 'BH' (src/conv2.cpp:105:6) in function 'conv2' more than one sub loop.&#xD;&#xA;Resolution: For help on HLS 200-960 see www.xilinx.com/cgi-bin/docs/rdoc?v=2023.1;t=hls+guidance;d=200-960.html" projectName="srcnn_hls" solutionName="solution1" date="2023-11-02T02:40:00.155+1100" type="Warning"/>
        <logs message="WARNING: [HLS 200-960] Cannot flatten loop 'TILE_ROW' (src/conv3.cpp:33:12) in function 'conv3' more than one sub loop.&#xD;&#xA;Resolution: For help on HLS 200-960 see www.xilinx.com/cgi-bin/docs/rdoc?v=2023.1;t=hls+guidance;d=200-960.html" projectName="srcnn_hls" solutionName="solution1" date="2023-11-02T02:40:00.147+1100" type="Warning"/>
        <logs message="WARNING: [HLS 200-960] Cannot flatten loop 'BH' (src/conv3.cpp:130:6) in function 'conv3' more than one sub loop.&#xD;&#xA;Resolution: For help on HLS 200-960 see www.xilinx.com/cgi-bin/docs/rdoc?v=2023.1;t=hls+guidance;d=200-960.html" projectName="srcnn_hls" solutionName="solution1" date="2023-11-02T02:40:00.140+1100" type="Warning"/>
        <logs message="WARNING: [HLS 200-960] Cannot flatten loop 'COL' (src/conv3.cpp:43:10) in function 'conv3' the outer loop is not a perfect loop because there is nontrivial logic in the loop latch.&#xD;&#xA;Resolution: For help on HLS 200-960 see www.xilinx.com/cgi-bin/docs/rdoc?v=2023.1;t=hls+guidance;d=200-960.html" projectName="srcnn_hls" solutionName="solution1" date="2023-11-02T02:40:00.122+1100" type="Warning"/>
        <logs message="WARNING: [HLS 200-960] Cannot flatten loop 'EXPORT' (src/conv1.cpp:133:10) in function 'export_output_buffer_c1' the outer loop is not a perfect loop because there is nontrivial logic before entering the inner loop.&#xD;&#xA;Resolution: For help on HLS 200-960 see www.xilinx.com/cgi-bin/docs/rdoc?v=2023.1;t=hls+guidance;d=200-960.html" projectName="srcnn_hls" solutionName="solution1" date="2023-11-02T02:40:00.115+1100" type="Warning"/>
        <logs message="WARNING: [HLS 200-960] Cannot flatten loop 'BH' (src/conv1.cpp:134:6) in function 'export_output_buffer_c1' more than one sub loop.&#xD;&#xA;Resolution: For help on HLS 200-960 see www.xilinx.com/cgi-bin/docs/rdoc?v=2023.1;t=hls+guidance;d=200-960.html" projectName="srcnn_hls" solutionName="solution1" date="2023-11-02T02:40:00.109+1100" type="Warning"/>
      </synLog>
      <simLog/>
      <mgLog/>
      <packageLog/>
      <csimLog/>
    </logs>
  </warningLogs>
</vivadoHLSLog:LogRoot>
