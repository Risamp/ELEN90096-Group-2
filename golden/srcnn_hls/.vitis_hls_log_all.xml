<?xml version="1.0" encoding="UTF-8"?>
<vivadoHLSLog:LogRoot xmlns:vivadoHLSLog="www.xilinx.com/vivadoHLSLog">
  <errorLogs>
    <logs>
      <csimLog>
        <logs message="ERROR: [SIM 211-100] CSim failed with errors." projectName="srcnn_hls" solutionName="solution1" date="2023-09-29T22:50:34.986+1000"/>
        <logs message="ERROR: [SIM 211-100] 'csim_design' failed: compilation error(s)." projectName="srcnn_hls" solutionName="solution1" date="2023-09-29T22:49:50.372+1000"/>
        <logs message="ERROR: [SIM 211-100] CSim failed with errors." projectName="srcnn_hls" solutionName="solution1" date="2023-09-29T22:48:43.751+1000"/>
        <logs message="ERROR: [SIM 211-100] CSim failed with errors." projectName="srcnn_hls" solutionName="solution1" date="2023-09-29T22:47:02.129+1000"/>
        <logs message="ERROR: [SIM 211-100] CSim file generation failed: compilation error(s)." projectName="srcnn_hls" solutionName="solution1" date="2023-09-29T22:40:48.129+1000"/>
        <logs message="ERROR: [SIM 211-100] CSim file generation failed: compilation error(s)." projectName="srcnn_hls" solutionName="solution1" date="2023-09-29T22:37:08.208+1000"/>
        <logs message="ERROR: [SIM 211-100] 'csim_design' failed: compilation error(s)." projectName="srcnn_hls" solutionName="solution1" date="2023-09-29T22:36:40.675+1000"/>
        <logs message="ERROR: [SIM 211-100] 'csim_design' failed: compilation error(s)." projectName="srcnn_hls" solutionName="solution1" date="2023-09-29T22:35:25.048+1000"/>
        <logs message="ERROR: [SIM 211-100] 'csim_design' failed: compilation error(s)." projectName="srcnn_hls" solutionName="solution1" date="2023-09-29T22:33:51.928+1000"/>
        <logs message="ERROR: [SIM 211-100] 'csim_design' failed: compilation error(s)." projectName="srcnn_hls" solutionName="solution1" date="2023-09-29T22:27:09.321+1000"/>
        <logs message="ERROR: [SIM 211-100] 'csim_design' failed: compilation error(s)." projectName="srcnn_hls" solutionName="solution1" date="2023-09-29T22:25:48.066+1000"/>
        <logs message="ERROR: [SIM 211-100] 'csim_design' failed: compilation error(s)." projectName="srcnn_hls" solutionName="solution1" date="2023-09-29T22:20:38.017+1000"/>
        <logs message="ERROR: [SIM 211-100] 'csim_design' failed: compilation error(s)." projectName="srcnn_hls" solutionName="solution1" date="2023-09-29T22:19:36.560+1000"/>
        <logs message="ERROR: [SIM 211-100] 'csim_design' failed: compilation error(s)." projectName="srcnn_hls" solutionName="solution1" date="2023-09-29T22:18:24.072+1000"/>
        <logs message="ERROR: [SIM 211-100] 'csim_design' failed: compilation error(s)." projectName="srcnn_hls" solutionName="solution1" date="2023-09-29T22:16:58.103+1000"/>
        <logs message="ERROR: [SIM 211-100] 'csim_design' failed: compilation error(s)." projectName="srcnn_hls" solutionName="solution1" date="2023-09-29T22:05:12.979+1000"/>
        <logs message="ERROR: [SIM 211-100] 'csim_design' failed: compilation error(s)." projectName="srcnn_hls" solutionName="solution1" date="2023-09-29T21:59:59.952+1000"/>
        <logs message="ERROR: [SIM 211-100] 'csim_design' failed: compilation error(s)." projectName="srcnn_hls" solutionName="solution1" date="2023-09-29T21:57:50.483+1000"/>
        <logs message="ERROR: [SIM 211-100] 'csim_design' failed: compilation error(s)." projectName="srcnn_hls" solutionName="solution1" date="2023-09-29T21:55:53.997+1000"/>
        <logs message="ERROR: [SIM 211-100] 'csim_design' failed: compilation error(s)." projectName="srcnn_hls" solutionName="solution1" date="2023-09-29T21:53:58.710+1000"/>
        <logs message="ERROR: [SIM 211-100] 'csim_design' failed: compilation error(s)." projectName="srcnn_hls" solutionName="solution1" date="2023-09-29T21:51:31.284+1000"/>
        <logs message="ERROR: [SIM 211-100] 'csim_design' failed: compilation error(s)." projectName="srcnn_hls" solutionName="solution1" date="2023-09-29T21:50:37.734+1000"/>
        <logs message="ERROR: [SIM 211-100] 'csim_design' failed: compilation error(s)." projectName="srcnn_hls" solutionName="solution1" date="2023-09-29T21:44:37.169+1000"/>
        <logs message="ERROR: [SIM 211-100] 'csim_design' failed: compilation error(s)." projectName="srcnn_hls" solutionName="solution1" date="2023-09-29T21:43:19.383+1000"/>
        <logs message="ERROR: [SIM 211-100] 'csim_design' failed: compilation error(s)." projectName="srcnn_hls" solutionName="solution1" date="2023-09-29T21:39:41.996+1000"/>
        <logs message="ERROR: [SIM 211-100] 'csim_design' failed: compilation error(s)." projectName="srcnn_hls" solutionName="solution1" date="2023-09-26T00:31:31.811+1000"/>
        <logs message="ERROR: [SIM 211-100] 'csim_design' failed: compilation error(s)." projectName="srcnn_hls" solutionName="solution1" date="2023-09-25T23:50:14.537+1000"/>
        <logs message="ERROR: [SIM 211-100] 'csim_design' failed: compilation error(s)." projectName="srcnn_hls" solutionName="solution1" date="2023-09-25T23:49:23.416+1000"/>
      </csimLog>
    </logs>
  </errorLogs>
  <warningLogs>
    <logs>
      <csimLog/>
    </logs>
  </warningLogs>
</vivadoHLSLog:LogRoot>
